tsl/test/sql/remote_connection_cache.sql

```diff
@@ -26,3 +26,6 @@ DO $d$
 $d$;
 
 SELECT _timescaledb_internal.test_remote_connection_cache();
+
+DROP DATABASE loopback_1;
+DROP DATABASE loopback_2;
```

Comment:
We started rewriting tests (e.g., `bgw_launcher`) create databases for the test using something like this:

```
\set TEST_DBNAME_2 :TEST_DBNAME _2
    .
    .
    .
CREATE DATABASE :TEST_DBNAME_2
```

You might want to consider switching to use this format.